BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -117,13 +197,13 @@ public final class ConvivaAnalytics: NSObject {
         }
     }
 
-    private func initSession() {
+    private func internalInitializeSession() {
         buildContentMetadata()
         sessionKey = client.createSession(with: contentMetadata)
         setupPlayerStateManager()
         updateSession()
 
-        if !isValidSession {
+        if !isSessionActive {
             logger.debugLog(message: "Something went wrong, could not obtain session key")
```

Comment:
There should be a `return`, shouldn't it?

Reply:
Actually there should be a throw that a SDK user will be notified about that. (Should never happen actually)

Reply:
🤦‍♂️